### DISCUSSION TITLE
LAN Server Hosting doesn't authenticate, even with RequireAuthentication="false"

### DISCUSSION THREAD

**Discussion by Chromaphasia** [FakeFishGames/Barotrauma#14343] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I hosted a dedicated server on my machine, and attempted to connect with another machine on the same network. The connecting machine got "Connection Authentication Failed" in response. I have set RequireAuthentication to false in the server settings and saved the file, but it doesn't seem to have worked, or if it has, it hasn't worked correctly.

### Reproduction steps

1. Start a dedicated server with RequireAuthentication = "False" in the serverSettings.xml
2. Attempt to connect with another machine using 'Direct Join' and inputting the first machine's IP. (In my case, this second machine does not have access to Steam/Epic Games but does have LAN access to the first machine)
3. Connection Authentication Failed error, but first machine can connect to server without issue (First machine does have access to steam however.)

Host Machine is Windows, Connecting Machine is Linux (Arch).

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted using a dedicated server

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Linux

**Comment by EdwardTheGamer** [comment]
I have this problem too...

**Comment by gerakis** [comment]
I also have this problem. Shouldnt the title be "always authenticates" rather than doesnt authenticate?

**Comment by Regalis11** [comment]
Thank you for the report! Based on a quick test, it appears that when using SteamP2P hosting, authentication cannot be disabled at the moment. It should be possible to work around the issue by disconnecting from Steam at the host's end. Regardless, we need to take a closer look at this and make sure it works on SteamP2P servers too - sorry for the inconvenience!

**Comment by Jasontti** [comment]
Tested a fix for this. No issues found.
Closing discussion as resolved.
